### PR TITLE
Add internal dependencies version to bom module

### PR DESCRIPTION
**Pull request by symphony-hong**
### Ticket
[APP-3078](https://perzoinc.atlassian.net/browse/APP-3078)

### Description
Please put here the intent of your pull request.

### Dependencies
List the other pull requests that should be merged before/along this one.

### Checklist
- [ ] Referenced a ticket in the PR title and in the corresponding section
- [ ] Filled properly the description and dependencies, if any
- [ ] Unit tests updated or added
- [ ] Javadoc added or updated
- [ ] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

**Comment by symphony-thibault** [comment]
You could link your current ticket as well for tracking purpose

File: symphony-bdk-bom/build.gradle [hunk 1]
```diff
@@ -19,8 +19,17 @@ dependencies {
 
     // define all our dependencies versions
     constraints {
+        // Internal modules dependencies (Keep them first)
+        api "com.symphony.platformsolutions:symphony-bdk-core:$project.version"
+        api "com.symphony.platformsolutions:symphony-bdk-http-api:$project.version"
+        api "com.symphony.platformsolutions:symphony-bdk-http-jersey2:$project.version"
```
Comment (symphony-thibault) [inline symphony-bdk-bom/build.gradle:25]: `symphony-bdk-http-webclient` is missing

**Review by symphony-youri — APPROVED** [review]
apart from Thibault's comment looks good

**Comment by symphony-hong** [comment]
Oops! I forgot to push the commit

**Pull request merged**